tests/parser/integration/NNCTests.cpp

```diff
@@ -220,6 +220,65 @@ EDITNNC
 /
 )";
 
+const std::string editnncr_input = R"(
+RUNSPEC
+
+OIL
+GAS
+WATER
+
+
+DIMENS
+   10 10  1  /
+
+GRID
+
+NNC
+   7 1 1 3 1 1 0.1 /
+   3 1 1 5 1 1 0.1 / 
+/
+
+
+DXV
+10*1000.0
+/
+
+DYV
+10*1000.0
+/
+
+DZ
+100*20.0
+/
+
+TOPS
+100*10
+/
+
+PORO
+   100*0.15 /
+
+EDIT
+
+EDITNNC
+5 1 1 3 1 1 2.0 / -- Will be removed after internalization as there is an NNC
+3 1 1 1 1 1 0.1 / -- stays internalized
+2 1 1 2 3 1 0.3 / -- Will be removed after internalization as there is an EDTNNCR
+/
+
+EDITNNCR
+8 1 1 3 1 1 2.0 /
+1 1 1 1 2 1 0.01 / -- This is ignored because the two cells are not ijk neighbours
```

Comment:
Is it not the opposite? I.e. ignored because they ARE ijk neighbours?

Reply:
Indeed. Fixed.